unittest/bindings/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(${PROJECT_NAME}_PYTHON_BINDINGS_TESTS
+    curves
+    constant
+    sinusoidal
+    minjerk
+    optimization
+    notebook
+    registration
+    constraints)
+
+foreach(TEST ${${PROJECT_NAME}_PYTHON_BINDINGS_TESTS})
+  python_build(. "test_${TEST}.py")
+  add_python_unit_test("test_pybinds_${TEST}"
+                       "unittest/bindings/test_${TEST}.py" python)
+  add_custom_target(
+    "test_pybinds_${TEST}"
+    ${CMAKE_COMMAND} -E env
+    PYTHONPATH=${PROJECT_BINARY_DIR}/python/ndcurves:$ENV{PYTHONPATH}
+    ${PYTHON_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/test_${TEST}.py")
```

Comment:
This is a lot of code to avoid a simple `PYTHONPATH=build/python python test/python/test_curves.py` or an even simpler `PYTHONPATH=build/python python -m unittest`, plus an eventual filter.

I don't think we should add all of this in all of  our projects. 

If your really need this feature, I'd suggest implementing this correctly in the cmake submodule `add_python_unit_test`

This is also wrong :
1. the `/ndcurves` shouldn't be there: we should be importing `ndcurves` in the tests, so PYTHONPATH should have the parent directory of the `ndcurves` folder
2. we shouldn't blindly append `$ENV{PYTHONPATH}`: if this is empty, you'll be only appending `:` at the end, which will append the current working directory in the search path and can cause unexpected behaviors.

Reply:
I implemented this command as I didn't know how to run them myself. I would say this is not as intuitive for others as you think. I still believe this is nice to have.

Putting this inside `add_python_unit_test` could be indeed beneficial, but it would require a global consensus. This will affect many more projects and I need other developers on my side. Would you support this?